### PR TITLE
Onboarding screen

### PR DESCRIPTION
**Pull request by pablomarti**
This PR adds the onboarding screen that should appear on the 1st time to the user.

Right now it is appearing every time the user goes into this page. In order to display it only once it will require to add NSUserDefaults maybe, but I didn't add it so it can be tested many times.

### CODE REVIEW

File: godtools/Views/Buttons/TransparentButton.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import UIKit
+
+class TransparentButton: GTButton {
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.cornerRadius = 5.0
+        self.borderColor = UIColor.gtWhite
+        self.borderWidth = 1.0
+        self.backgroundColor = UIColor.clear
```
Comment (ryancarlson) [inline godtools/Views/Buttons/TransparentButton.swift]: = .clear

File: godtools/Views/Buttons/TransparentButton.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+//
+
+import UIKit
+
+class TransparentButton: GTButton {
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.cornerRadius = 5.0
+        self.borderColor = UIColor.gtWhite
```
Comment (ryancarlson) [inline godtools/Views/Buttons/TransparentButton.swift]: = .gtWhite

File: godtools/Views/Labels/GTLabel.swift [hunk 3]
```diff
@@ -14,6 +14,9 @@ class GTLabel: UILabel {
     @IBInspectable var gtStyle: String = "" {
         didSet {
             switch self.gtStyle {
+            case "whiteTitle":
+                self.font = UIFont.gtLight(size: 28.0)
+                self.textColor = UIColor.gtWhite
```
Comment (ryancarlson) [inline godtools/Views/Labels/GTLabel.swift]: .gtWhite

File: godtools/ViewControllers/Platform/OnboardingViewController.swift [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    fileprivate func initialSetup() {
+        let viewWidth = self.view.frame.width
+        self.page2View.transform = CGAffineTransform(translationX: viewWidth, y: 0)
+        self.pageControl.transform = CGAffineTransform(scaleX: 2.0, y: 2.0);
+    }
+    
+    @objc fileprivate func handleGesture(gesture: UISwipeGestureRecognizer) {
+        if gesture.direction == UISwipeGestureRecognizerDirection.right {
+            displayPage1()
+        } else if gesture.direction == UISwipeGestureRecognizerDirection.left {
```
Comment (ryancarlson) [inline godtools/ViewControllers/Platform/OnboardingViewController.swift]: probably just ".left" here?

File: godtools/ViewControllers/Platform/OnboardingViewController.swift [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    }
+    
+    fileprivate func initialSetup() {
+        let viewWidth = self.view.frame.width
+        self.page2View.transform = CGAffineTransform(translationX: viewWidth, y: 0)
+        self.pageControl.transform = CGAffineTransform(scaleX: 2.0, y: 2.0);
+    }
+    
+    @objc fileprivate func handleGesture(gesture: UISwipeGestureRecognizer) {
+        if gesture.direction == UISwipeGestureRecognizerDirection.right {
```
Comment (ryancarlson) [inline godtools/ViewControllers/Platform/OnboardingViewController.swift]: probably just ".right" here?

**Comment by pablomarti** [comment]
@ryancarlson I just uploaded the changes requested, thanks!

**Pull request merged**